### DISCUSSION TITLE
Android build with a custom web path

### DISCUSSION THREAD

**Discussion by ram-if** [ionic-team/capacitor#3341] [Q&A]
Hi All

I am relatively new to Capacitor..  I followed the capacitor documentation and managed to build an android app from my web application (which was developed using ReactJS).

My web application has multiple paths (called "routes" in React)
https://www.example.com/
https://www.example.com/signin
https://www.example.com/signup
.
.
.

When I open the android app, it is by default showing the root path (https://www.example.com)

Is it possible to show a different path in Android by default (https://www.example.com/signin)  instead of the root path?

When I open the android application, I basically want the user to be shown the Sign In screen instead of to the info page in root path.. Is there a way to configure the custom web path in capacitor build settings?

Thanks in advance!
Ram

**Comment by imhoffd** [comment]
There isn't a configuration option to load a different path in Capacitor. You'll need to use conditional [Redirect](https://reactrouter.com/web/api/Redirect) rules (assuming you're using React Router). 

```jsx
<Route exact path="/">
  {loggedIn ? <HomePage /> : <Redirect to="/signin" />}
</Route>
```
